dbt_platform_helper/utils/versioning.py

```diff
@@ -245,7 +245,9 @@ def check_platform_helper_version_mismatch():
 
```

Comment:
@WillGibson Do you know why this check is here? It seems to work fine when running locally with this check commented out. It means there's some functionality which can't be tested manually before releasing which isn't ideal.

Reply:
It is to stop it messing up when you are using a locally built version of `platform-helper`, not one installed from PyPi.

E.g. in the regression tests.

Reply:
I added a test around this check.